server/utils/isServiceEnabled.ts

```diff
@@ -0,0 +1,5 @@
+import { HeaderFooterMeta } from '../data/interfaces/component'
+
+export default (serviceId: string, componentsMeta: HeaderFooterMeta | undefined): boolean => {
+  return !!componentsMeta?.services?.find(service => service.id === serviceId)
```

Comment:
Is this only finding whether the `activities` service exists in the metadata, rather than also checking that the current caseload is also in the list?

Reply:
The header has already done that work and made the decision if activities should be shown to the user. This is using that decision